packages/Jms/tests/Rector/Property/JmsInjectAnnotationRector/Wrong/wrong.php.inc

```diff
@@ -7,17 +7,17 @@ use JMS\DiExtraBundle\Annotation as DI;
 class SomeController
 {
     /**
-     * @DI\Inject("entity.manager")
+     * @DI\Inject("Rector\Symfony\Tests\Rector\FrameworkBundle\AbstractToConstructorInjectionRectorSource\SomeEntityManager")
      */
-    private $entityManager;
+    public $entityManager;
```

Comment:
For public → private use new standalone case - `wrong4.php.inc` → `correct4.php.inc`, 

and put it here: https://github.com/rectorphp/rector/blob/173cb5a26cb2a6bf880080351e4b9adf8b0a9d71/packages/Jms/tests/Rector/Property/JmsInjectAnnotationRector/JmsInjectAnnotationRectorTest.php#L25